### DISCUSSION TITLE
raw.githubusercontent.com is not working

### DISCUSSION THREAD

**Discussion by GotAJadeInZooba** [community/community#67595] [Actions]
### Select Topic Area

Bug

### Body

Whenever I try to open any files on raw.githubusercontent by clicking the raw button. it just doesnt load and says "it took too long to respond" how do i fix that?

**Comment by shreya-oeo** [comment]
use dns server of 1.1.1.1 ( 1dot1dot1dot1.cloudflare-dns.com ) as private dns or warp application.

**Comment by priyanujgogoi-28** [comment]
Most probably, your ISP might have blocked `raw.githubusercontent.com` site. You can try these workarounds & see if it helps:

- Using a different ISP
- Using a VPN
- Changing your DNS
